crates/bevy_asset/src/direct_access_ext.rs

```diff
@@ -9,6 +11,9 @@ pub trait DirectAssetAccessExt {
     /// Insert an asset similarly to [`Assets::add`].
     fn add_asset<A: Asset>(&mut self, asset: impl Into<A>) -> Handle<A>;
 
+    /// Insert an [`Arc`]ed asset similarly to [`Assets::add_arc`].
+    fn add_arc_asset<A: Asset>(&mut self, asset: impl Into<Arc<A>>) -> Handle<A>;
```

Comment:
I think I'd prefer `Into<Arc<A>>` because it allows for other types to be used like for example `Box<A>`. It definitely could be `Arc<A>`, but I'd prefer to err on the side of caution here.

Reply:
Fair enough